js/okex.js

```diff
@@ -3678,6 +3678,14 @@ module.exports = class okex extends Exchange {
         return await this.modifyMarginHelper (symbol, amount, 'add', params);
     }
 
+    isDemoAccount (enable) {
```

Comment:
I suggest that we stick to the unified API and override setSandboxMode instead.

Reply:
@kroitor  I think there is some problem.
The 'testnet' you referred, is of V3 api: https://www.okex.com/docs/en/#3added-testing-environment-services%EF%BC%9A

however, with V5 api, there is no testnet, instead they do use 'Demo Trading' mode & api. 

https://www.okex.com/docs-v5/en/#overview-demo-trading-services ( tutorial here:  https://www.okex.com/academy/en/how-to-practice-trading-crypto-on-okex-with-demo-trading/ )

So, I think testnet wont work in this case, and we should use the V5 approach. What do you suggest?

Reply:
I've changed the name to 'setSandboxMode'. I've tested and it works on my side as expected.